common/wireaddr.c

```diff
@@ -1,17 +1,25 @@
 #include <arpa/inet.h>
 #include <assert.h>
+#include <ccan/io/io.h>
+#include <ccan/str/hex/hex.h>
 #include <ccan/tal/str/str.h>
+#include <common/base32.h>
 #include <common/type_to_string.h>
 #include <common/utils.h>
 #include <common/wireaddr.h>
+#include <errno.h>
+#include <fcntl.h>
 #include <netdb.h>
 #include <netinet/in.h>
 #include <sys/socket.h>
 #include <sys/types.h>
+#include <unistd.h>
 #include <wire/wire.h>
 
+
+
 /* Returns false if we didn't parse it, and *cursor == NULL if malformed. */
-bool fromwire_wireaddr(const u8 **cursor, size_t *max, struct wireaddr *addr)
+bool fromwire_wireaddr(const u8 ** cursor, size_t * max, struct wireaddr * addr)
```

Comment:
Editor settings maybe? I think it wouldn't make sense if two people had different editor settings and would alternate on committing to the same file while always undoing the automated editor changes. If there is some rationale behind this, please describe this change. Thanks.